templates/_media_upload_tab.tpl

```diff
@@ -1,3 +1,5 @@
-<li {% if tab == "multi" %}class="active"{% endif %}>
-    <a data-toggle="tab" href="#{{ tab }}-multi">{_ Multi upload _}</a>
-</li>
+{% if id or not m.acl.is_allowed.use.mod_admin_multiupload %}
```

Comment:
Should that `not` not be removed?

I also assume that this should not be dependent on the presence of `id` so maybe:

```erlang
{% if m.acl.is_allowed.use.mod_admin_multiupload %}
```

Reply:
I agree